scripts/i18n.mjs

```diff
@@ -11,7 +11,8 @@ const PATH_TO_OUTPUT = join(PATH_FROM_ROOT, 'lib', 'types', 'i18n.d.ts');
  * Generates TypeScript interfaces from the English translation file.
  */
 const generateTypes = async () => {
-	const { default: en } = await import(PATH_TO_EN_JSON, { assert: { type: 'json' } });
+	// https://github.com/tc39/proposal-import-attributes
```

Comment:
?

Reply:
my mistake... this shouldn't have been here